iceoryx_posh/source/version/version_info.cpp

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#include "iceoryx_posh/version/version_info.hpp"
+#include "iceoryx_utils/cxx/convert.hpp"
+
+#include <algorithm>
+
+namespace iox
+{
+namespace version
+{
+VersionInfo::VersionInfo(const uint16_t versionMajor,
+                         const uint16_t versionMinor,
+                         const uint16_t versionPatch,
+                         const uint16_t versionTweak,
+                         const BuildDateStringType& buildDateString,
+                         const CommitIdStringType& commitIdString) noexcept
+    : m_versionMajor(versionMajor)
+    , m_versionMinor(versionMinor)
+    , m_versionPatch(versionPatch)
+    , m_versionTweak(versionTweak)
+    , m_buildDateString(buildDateString)
+    , m_commitIdString(commitIdString)
+{
+}
+
+VersionInfo::VersionInfo(const cxx::Serialization& serial) noexcept
+{
+    cxx::string<100> tmp_m_buildDateString;
+    cxx::string<100> tmp_commitIdString;
```

Comment:
Could you please use the defined `CommitIdStringType ` instead of `cxx::string<100>`